_posts/2019-01-16-lecture-02.md

```diff
@@ -0,0 +1,459 @@
+---
+layout: distill
+title: "Lecture 02: Bayesian Networks"
+description: Overview of Bayesian Networks, their properties, and how they can  be helpful to model the joint probability distribution over a set of random variables. 
+date: 2019-01-16
+
+lecturers:
+  - name: Eric Xing
+    url: "https://www.cs.cmu.edu/~epxing/"
+
+authors:
+  - name: Cathy Su
+    url: "https://ceesu.github.io"   
+  - name: Angel C. Hernandez
+    url: "https://github.com/ahernandez105"
+  - name: Mark Cheung
+    url: "#"
+  - name: Xueqian Li
+    url: "#"
+
+editors:
+  - name: Editor 1  # editor's full name
+    url: "#"  # optional URL to the editor's homepage
+
+abstract: >
+---
+
+# Readings
```

Comment:
Please do not explicitly split your notes into `Readings` and `In class notes` parts, because it becomes repetitive and breaks the flow. Please integrate the `Readings` part as additional points into the `In class notes` part.

Reply:
Hi Maruan, thanks for your review. I added the other changes which were requested from my section. However I think it could be confusing for people who are looking between the lecture notes and the slides if the readings and notes are integrated, since the material doesn't totally overlap. Would it be acceptable to put the readings after the notes, and/or to have a table of contents at the beginning?

Reply:
Sure, you can move readings to the end, after the lecture notes. Just remove the heading `In class notes` (that's assumed by default) and rename (`Readings` to `Notes from reading materials`).